posts/2024-07-02-24.0.0.7-beta.adoc

```diff
@@ -0,0 +1,243 @@
+
+This update ensures that operations teams responsible for monitoring, can now have a detailed and comprehensive view of HTTP requests, enhancing the monitoring and management capabilities within Open Liberty environments.
+
+Previously, Open Liberty tracked servlet and JAX-RS/RESTful resource requests to the server as separate MBeans (servlet MBeans and REST MBeans). This data might then be forwarded to the mpMetrics feature to create corresponding metrics. Monitoring tools like Prometheus and Grafana could visualize this metric data to showcase server performance and responsiveness.
+
+With the new update, HTTP connections are now tracked more comprehensively, combining request details into a unified `HttpStatsMXBean`. This approach simplifies monitoring and provides a more cohesive view of server performance, making it easier for operations teams to ensure optimal server responsiveness.
+
+This enhancement is an auto-feature that activates with `monitor-1.0` and any feature that uses the servlet engine, currently supporting JEE 10 features like `servlet-6.0`, `pages-3.1`, and `restfulWS-3.1`. It also supports registering HTTP metrics with `mpMetrics-5.x` features. To enable this function, add the following to your `server.xml` file:
+
+[source,xml]
+----
+<featureManager>
+    <feature>monitor-1.0</feature>
+    <feature>servlet-6.0</feature>
+    <feature>mpMetrics-5.0</feature>
+</featureManager>
+----
+
+
+For detailed configuration, see the link:https://openliberty.io/docs/latest/microprofile-config-properties.html#metrics[MicroProfile Config Properties page]. 
+
+For more information about the HTTP semantic convention used, see the link:https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#metric-httpserverrequestduration[Open Telemetry HTTP semantic convention] page.
+    
+    
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/28603
+// Contact/Reviewer: volosied,pnicolucci
+// // // // // // // // 
+[#faces41]
+== Jakarta EE11 Faces 4.1
+
+Jakarta Faces is a Model-View-Controller (MVC) framework for building web applications, offering many convenient features such as state management and input validation. The Jakarta Faces 4.1 (`faces-4.1`) update introduces a range of incremental improvements that are aimed at enhancing the developer experience. These enhancements include increased use of generics in FacesMessages and other locations. Improved CDI integration now requires events for `@Initialized`, `@BeforeDestroyed`, and `@Destroyed` for built-in scopes like FlowScoped and ViewScoped, and it allows injecting the current Flow into a backing bean. Also, a new UUIDConverter is added to the API, simplifying the handling of UUIDs.
```

Comment:
```suggestion
Jakarta Faces is a Model-View-Controller (MVC) framework for building web applications, offering many convenient features, such as state management and input validation. The Jakarta Faces 4.1 (`faces-4.1`) update introduces a range of incremental improvements to enhance the developer experience. These enhancements include increased use of generics in `FacesMessages` and other locations. Improved CDI integration now requires events for `@Initialized`, `@BeforeDestroyed`, and `@Destroyed` for built-in scopes like `FlowScoped` and `ViewScoped`, and it allows injecting the current `Flow` into a backing bean. Also, a new `UUIDConverter` is added to the API, which simplifies the handling of UUIDs.
```

Reply:
+1 for above. 

And can you reword this line? 
`Improved CDI integration now requires events` -> `Improved CDI integration **by requiring** events`